app-crypt/tpm-tools/files/tpm-tools-1.3.9.1-openssl-1.1.patch

```diff
@@ -0,0 +1,106 @@
+Author: Philipp Kern <pkern@debian.org>
+Subject: Fix openssl1.1 support in data_mgmt
+Date: Tue, 31 Jan 2017 22:40:10 +0100
+
+
```

Comment:
please add URL to open bug at upstream where patch is attached so we can followup

Reply:
Where is none. Will open one.